include/priv/instr.h

```diff
@@ -144,8 +144,8 @@ typedef enum _InstrType {
     IT_CMOVS, IT_CMOVNS, IT_CMOVP, IT_CMOVNP,
     IT_CMOVL, IT_CMOVGE, IT_CMOVLE, IT_CMOVG,
 
-    IT_SETO, IT_SETNO, IT_SETC, IT_SETNC, IT_SETZ, IT_SETNZ, IT_SETBE, IT_SETA,
-    IT_SETS, IT_SETNS, IT_SETP, IT_SETNP, IT_SETL, IT_SETGE, IT_SETLE, IT_SETG,
+    IT_SETA, IT_SETAE, IT_SETB, IT_SETBE, IT_SETE, IT_SETG, IT_SETGE, IT_SETL,
```

Comment:
Not in DBrew directly, but in the LLVM back-end (see [llvm/src/llflags.c:60](https://github.com/aengelke/dbrew/blob/llvm/llvm/src/llflags.c#L60)). This system could also be used in DBrew to avoid redundant condition computation for `setcc`, `cmovcc` and `jcc`. I agree that a comment would be good.